docs/installation/index.md

```diff
@@ -18,19 +18,36 @@ The Autoware Foundation provides no support on other platforms than those listed
 
 Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system dependencies.
 
-## Installation steps of Autoware
+## Installation of Autoware
 
-See [here](autoware) for how to install Autoware.
+There are two ways to set up Autoware. Choose one according to your preference.
 
-## Installation steps of tools for users
+### 1. Docker installation
+
+Docker can ensure that all developers in a project have a common, consistent development environment.
+It is recommended for beginners, light users, people who do not use Ubuntu.
+
+See [here](https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/installation/autoware/docker-installation.md) for the detailed steps.  
+Refer to [autowarefoundation/autoware/docker/README.md](https://github.com/autowarefoundation/autoware/blob/main/docker/README.md) as well.
+
+### 2. Source installation
+
+Source installation is for the cases where more granular control of the installation environment is needed.
+It is recommended for skilled users or people who want to customize the environment.  
+Note that some problems may occur depending on your local environment.
```

Comment:
@kenji-miyake Although in theory it is good to have this line as cover in case someone's environment breaks after installing Autoware, in practice I feel that it is too vague to be helpful. It also suggests that the installation process is not stable which is not true!

Could we remove this line perhaps?

Reply:
@LalithVipulananthan How about writing like "Refer to the troubleshooting page when you occur a problem."?

Reply:
This seems like a good compromise. Since installation problems could occur with Docker as well, could we add this suggestion further up the page around line 23? For example:

> There are two ways to set up Autoware. Choose one according to your preference. 
> If any issues occur during installation, refer to the Troubleshooting guide.

Reply:
Yes, it's nice. :+1: 
@shmpwk Could you please update the documents as per Lalith-san's suggestion? :pray:

Reply:
StricIy speaking, "support page" is better than troubleshooting page https://autowarefoundation.github.io/autoware-documentation/main/support/,  and I've fixed it.

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/104/commits/38068a960e1ced9ee0c0aebe2a9bb83980075d51